### DISCUSSION TITLE
how to add version php 7 beside to php version 8

### DISCUSSION THREAD

**Discussion by abdelrahmanmohsen123** [leokhoa/laragon#507] [General]
how to add version php 7 beside to php version 8

**Comment by eagletrance** [comment]
Just go and download the PHP version you require, paste the additional version in laragon\bin\php\ laragon will pick it up.

You can then hotswap between them with the laragon interface ( restarting the server ).

**Reply by panjikmt** [reply]
Thank you, it's working.

Don't forget to download the **Windows version** so it can works on Windows.

**Reply by JulioCDias** [reply]
Thank you!

**Comment by abdelrahmanmohsen123** [comment]
i already do that but unfortunately doesnt work


‫في الخميس، 18 مايو 2023 في 2:14 م تمت كتابة ما يلي بواسطة ‪Alex G‬‏ <‪
***@***.***‬‏>:‬

**Comment by eagletrance** [comment]
Can you describe more on how it doesn't work? 

As in, laragon doesn't allow you to switch to it, server won't start  etc. etc.

![image](https://github.com/leokhoa/laragon/assets/18198786/75646d2b-eb4c-4d5c-8687-b3dc0dff0453)
